scenes/karma.conf.js

```diff
@@ -25,8 +37,14 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: [isBazel ? 'ChromeHeadlessNoSandbox' : 'ChromeHeadlessLocal'],
     singleRun: false,
+    customLaunchers: customLaunchers,
     restartOnFileChange: true
   });
+
+
+  if (process.env['TRAVIS']) {
```

Comment:
I don't think we use Travis. Is this an artifact from some testing?

Reply:
No I'm not sure, it already existed in all the other karma configs

Reply:
Interesting, seems like a legacy artifact. We can clean-up ourselves, but this config did not have this travis check before, so we shouldn't add it here?

Reply:
I think I was aligning this with karma.conf.js (in the root), I don't recall why 😬